### ISSUE TITLE
URL in [...](link) gets Double Linked

### ISSUE THREAD

**Issue by donatj** [erusev/parsedown#417]
If you have a URL in your content to be linked, it comes out double linked.

For example

``` markdown
[https://example.com](http://local.example.com)
```

outputs wrapped in two `<a>` tags:

``` html
<p><a href="http://local.example.com"><a href="https://example.com">https://example.com</a></a> </p>
```

Whereas in other parsers including Github's (ex: [https://example.com](http://local.example.com) ) it comes out:

``` html
<p><a href="http://local.example.com">https://example.com</a> </p>
```

**Comment by PhrozenByte** [comment]
dup of #358

**Comment by aidantwoods** [comment]
Fixed in #433
